static/styles/compose.css

```diff
@@ -228,26 +228,32 @@
 
     button {
         background: transparent;
-        font-size: 20px;
-        font-weight: bold;
-        line-height: 20px;
         opacity: 0.6;
-        border: 0;
-        padding: 0;
         margin-left: 4px;
-        vertical-align: unset;
+        border: 0;
 
         &:hover {
             opacity: 1;
         }
     }
 }
 
-.collapse_composebox_button,
-#compose_close {
+.collapse_composebox_button {
     display: none;
 }
 
+.composebox-arrow {
```

Comment:
Do we need this new class?  It seems like `.expand_composebox_button, .collapse_composebox_button` might be cleaner.  Or if we want it, naming it `.composebox-fullsize-toggle` would be more descriptive.